js/main.js

```diff
@@ -101,3 +104,69 @@ var addPinToDom = function () {
 
 addPinToDom();
 
+// Вернуть тип отеля для карточки отеля с описанием
+var ROOM_TYPES = {
+  flat: 'Квартира',
+  bungalo: 'Бунгало',
+  house: 'Дом',
+  palace: 'Дворец'
+};
+
+// Создать шаблон card
+var createMapCard = function (obj) {
+  var mapFilterContainerNode = document.querySelector('.map__filters-container');
+  var cardNode = document.querySelector('#card').content;
+  var popup = cardNode.querySelector('.popup').cloneNode(true);
+
+  popup.querySelector('.popup__title').textContent = obj.offer.title;
+  popup.querySelector('.popup__text--address').textContent = obj.offer.address;
+  popup.querySelector('.popup__text--price').textContent = obj.offer.price + '₽/ночь';
+  popup.querySelector('.popup__type').textContent = ROOM_TYPES[obj.offer.type];
+  popup.querySelector('.popup__text--capacity').textContent = obj.offer.rooms + ' комнаты для ' + obj.offer.guests + ' гостей';
+  popup.querySelector('.popup__text--time').textContent = 'Заезд после ' + obj.offer.checkin + ', выезд до ' + obj.offer.checkout;
+  popup.querySelector('.popup__description').textContent = obj.offer.description;
+  popup.querySelector('.popup__avatar').src = obj.author.avatar;
+  var popupPhotos = popup.querySelector('.popup__photos');
+  var popupPhotosImg = popupPhotos.querySelector('img');
```

Comment:
Лучше всегда искать через классовый селектор если есть такая возможность
```suggestion
  var popupPhotosImg = popupPhotos.querySelector('.popup__photo');
```